Rakefile

```diff
@@ -50,9 +50,22 @@ end
 task :nylas_build do
   Jeweler::Tasks.new do |gem|
     setup_nylas_gem(gem)
+    gem.platform = 'java'
```

Comment:
Yeah, I did not mean to commit that. Still need to finish up the work for publishing separate gems. 

Publishing a JRuby gem is the same as MRI, the difference is you need to set the platform. The work flow wrinkle is you have to build the gem while in the correct Ruby,e.g. rbenv switch to MRI than switch to JRuby to publish both gems. This can be wrapped up in a shell script.